release-notes/v1_71.md

```diff
@@ -20,6 +20,20 @@ Until the August milestone release notes are available, you can still track our
 
 We really appreciate people trying our new features as soon as they are ready, so check back here often and learn what's new.
 
+## Workbench
+
+### More codecs support
+
+We now ship more codecs as part of our Electron build. This will allow more audio and video files to be played from Notebooks, from your code, or by extensions embedding audio and video in webviews. The full list of codecs and codecs containers that we support is:
```

Comment:
Given how we have never officially documented the codecs supported by the application, maybe the following is less vague. Feel free to tweak the wording :)

```suggestion
### FFmpeg codecs support

FFmpeg shared library shipped as part of the application previously had only support for `FLAC` codec, with `1.71` the library has been updated to support the following list of codecs and containers. This will allow more audio and video files to be played from Notebooks or by extensions embedding audio and video in webviews.
```